app/src/main/res/values-nl/strings.xml

```diff
@@ -71,11 +71,11 @@
     <string name="invalid_image_path">Ongeldig pad naar afbeelding</string>
     <string name="image_editing_failed">Fout bij bewerken van afbeelding</string>
     <string name="edit_image_with">Afbeelding bewerken met:</string>
-    <string name="no_editor_found">Geen app gevonden voor bewerking</string>
+    <string name="no_editor_found">Geen bewerk-app gevonden</string>
     <string name="unknown_file_location">Onbekende bestandslocatie</string>
-    <string name="error_saving_file">Kan het bronbestand niet overschrijven</string>
-    <string name="rotate_left">Linksom draaien</string>
-    <string name="rotate_right">Rechtsom draaien</string>
+    <string name="error_saving_file">Het bronbestand kan niet worden overschreven</string>
+    <string name="rotate_left">Naar links draaien</string>
```

Comment:
No

Reply:
Yes. Here in the Netherlands, this is how we say it and how it's translated. I've never seen anything translated with "Linksom draaien", except in Belgium.

Reply:
Lolz, "tegenwijzerzin" would be Flemish, yeah, not "linksom".

"Tegenwijzerzin (België) oftewel linksom (Nederland) is een manier om een draairichting aan te geven, in dit geval tegengesteld aan de draaizin van een klok: 12 → 9 → 6 → 3 → 12."

Reply:
Oké, maar dan nog zie ik dat de meeste software (die ik gebruik) "naar links draaien" gebruikt en niet "linksom draaien". "naar ... draaien" is dus gebruikelijker.

Reply:
ok